### PR TITLE
Have done changes on baba la gumbala lyric with instructions

### PR DESCRIPTION
**Pull request by shiundumichael**

### CODE REVIEW

File: src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts [hunk 1]
```diff
@@ -415,7 +415,7 @@ export const Baba_La_Gumbala_Instructions: Song = {
             "endTimeMinute": "1",
             "endTimeSecond": "37.59"
         },
-        {
+  6      {
```
Comment (michaelclapham) [inline src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:418]: This is causing a compile error. Please remove this 6

**Comment by michaelclapham** [comment]
Around 1:23 Baba la gumbala gumbala havise needs to come in a few seconds earlier.

**Comment by michaelclapham** [comment]
At around 1:27 Hanana nana havise needs to come in sooner

**Comment by michaelclapham** [comment]
At 1:49 the instruction 
_Add fun actions like making a big pot of stone soup,chopping the ingredients,stirring pot and testing the soup_
should be
Add fun actions, like making a big pot of stone soup, chopping the ingredients, stirring the pot and tasting the soup.

File: src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts (hunk 1 above)
Comment (barbra2020) [reply src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts]: @shiundumichael  
have you removed the 6

**Comment by shiundumichael** [comment]
@barbra2020 
Have removed the 6 but am yet to do more corrections

**Comment by maxwellfundi** [comment]
@shiundumichael  You addresses some of these issues but you have not finished all the issues raised by mike specifically
1. The Hanana nana havise at 1.27 needs to come earlier say 125.9 
2. Third instruction  on 1.42 the last part should be tasting the soup not testing the soup
3. The first instruction at the start says one, play...... the other instructions need to have that part too.

**Comment by shiundumichael** [comment]
Am closing this because it has same changes as PR #45

**Pull request closed without merging**